tests/cv/test_gkp.py

```diff
@@ -23,9 +25,11 @@
 
 
 N = 50
+now = datetime.now()
+int_time = int(str(now.year) + str(now.month) + str(now.day) + str(now.hour) + str(now.minute))
 
 # Construct random numbers from an integer and fractional part.
-alpha_vals = np.append(np.random.rand(5) * 5, np.sqrt(np.pi))
+alpha_vals = np.append(rng(int_time).random(5) * 5, np.sqrt(np.pi))
```

Comment:
Why not relying on pytest's random seeding [`pytest-randomly`](https://pypi.org/project/pytest-randomly/)? Otherwise it'd be good to log the `int_time` to be able to execute the tests with the same seed in case needed.

Reply:
Yeah, I forgot to log the integer seed. I'll do that.